test/TestCase.cpp

```diff
@@ -201,6 +201,8 @@ bool EVMVersionRestrictedTestCase::validateSettings(langutil::EVMVersion _evmVer
 	else if (comparator == "!")
 		return !(_evmVersion == version);
 	else
-		throw runtime_error("Invalid EVM comparator: \"" + comparator + "\"");
-	return false; // not reached
+	{
+		BOOST_THROW_EXCEPTION(runtime_error{"Invalid EVM comparator: \"" + comparator + "\""});
+		return false; // not reached
```

Comment:
Can we remove this comment? :-)

Reply:
Good question. Sure I can, wasn't part of my PR (I just indented). Hold tight.